platform/linuxbsd/display_server_x11.cpp

```diff
@@ -126,7 +126,7 @@ bool DisplayServerX11::has_feature(Feature p_feature) const {
 		case FEATURE_WINDOW_TRANSPARENCY:
 		//case FEATURE_HIDPI:
 		case FEATURE_ICON:
-		case FEATURE_NATIVE_ICON:
+		//case FEATURE_NATIVE_ICON:
```

Comment:
Could this line just be removed entirely? Is there a benefit to commenting it out?

Reply:
Other lines were commented out, but I'm not sure why. Does it hint at a future implementation being possible? As bruvzg outlined above, it probably makes no sense to support this feature on Linux.